app/javascript/components/editor/TestRunSummary.tsx

```diff
@@ -1,9 +1,29 @@
-import React, { useState, useEffect, useCallback, useRef } from 'react'
-import { Submission, TestRun, TestRunStatus } from './types'
+import React, { useEffect, useCallback, useRef } from 'react'
+import { TestRun, TestRunStatus } from './types'
 import { TestRunChannel } from '../../channels/testRunChannel'
 import { TestRunSummaryContent } from './TestRunSummaryContent'
 import { fetchJSON } from '../../utils/fetch-json'
 
+const TestRunSummaryHeader = ({ testRun }: { testRun: TestRun }) => {
+  switch (testRun.status) {
+    case TestRunStatus.FAIL:
+      return (
+        <div className="summary-status failed">
+          <span className="--dot" />1 test failure
```

Comment:
In theory, there could be multiple test failures. Do we always want to list it as one failure?

Reply:
We show all the rest as skipped.

Reply:
Okay!